webapp/src/serial.tsx

```diff
@@ -21,8 +21,6 @@ export class Editor extends srceditor.Editor {
     maxConsoleLineLength: number = 500
     maxConsoleEntries: number = 100
     active: boolean = true
-    rawDataBuffer: string = ""
-    maxBufferLength: number = 5000
```

Comment:
any reason why we are dropping this?

Reply:
It was only a placeholder until exportToCSV was implemented.  Now that you've done that, we don't need this.